File: python/pad_array.py
#REMEMBER TO PSEUDOCODE
def pad(array, min_size, value = None):
    """
    Given an array, a min_size, and a value, if the length
    of the array is less than min_size, append value intil min
    size is reached.
    """
    if len(array) >= min_size: #don't need to add anything
        return array
    
    new_array = array.copy()
    size_diff = min_size - len(array)
    for i in range(size_diff):
        new_array.append(value)
    
    return new_array

File: js/padArray.js
// REMEMBER TO PSEUDOCODE
exports.pad = (array, minSize, value=null) => {
    //don't need to add if minSize is less than or equal to array size
    if (minSize <= array.lenth) {
        return array
    }
    const newArray = [...array] //copy
    diffSize = minSize - array.length
    for (let i=0; i< diffSize; i++) {
        newArray.push(value)
    }
    //push the difference between the minsize and array 
    return newArray

}
